packages/zowe-explorer/src/Profiles.ts

```diff
@@ -582,20 +567,7 @@ export class Profiles extends ProfilesCache {
                     }
                     break;
                 case "port":
-                    if (schemaValues[value] === undefined) {
```

Comment:
Maybe we change our dialog validation instead?

Now, I can specify only a hostname in the dialogs, without any port and everything will be fine, until the whole profile will be created. And I can get an error during profile validation (which is not really clear in some cases).

We have `validateAndParseUrl` method, which we can use to cover the bug (or a feature) inside the Node js URL constructor. Maybe we can add an additional check there? Like, if the protocol is equal to `https`, and the port is `0`, than we can assume, the port is `443` - default one. We can cover it by some unit tests and it will not be a surprise.

What do you think?

Reply:
There is also a super related issue (about `443` port and `https` protocol combination). I think, proper validation will fix it as well.

How to reproduce: once I specified a `443` port and a `https` protocol during profile creation, I do not see the correct profile value during profile **updating**.

Reply:
Hey, is there an issue already open for this 443 port/https issue? If so could you post the link here? Then we could close both with this PR possibly.

Reply:
Nope, I just played with it and identified :)